blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_1.py

```diff
@@ -0,0 +1,130 @@
+"""Formula 5.1 from NEN-EN 1992-1-1+C2:2011: Chapter Structural Analysis."""
+# pylint: disable=arguments-differ
+import numpy as np
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import M
+
+
+class Form5Dot1Imperfections(Formula):
+    """Class representing formula 5.1 for the calculation of initial inclination imperfections, Θi."""
+
+    label = "5.1"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        theta_0: float,
```

Comment:
That's a great idea. Let's add 'unitless' as 'UNITLESS' (written in full 😄, since I don't know a shorthand for this one). Clarity for the user above all!

Reply:
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/43a2c69a-f5ee-4519-b5c4-03cf41784cd9)